third_party/terraform/utils/compute_instance_helpers.go

```diff
@@ -329,3 +329,21 @@ func flattenShieldedVmConfig(shieldedVmConfig *computeBeta.ShieldedVmConfig) []m
 		"enable_integrity_monitoring": shieldedVmConfig.EnableIntegrityMonitoring,
 	}}
 }
+
+func expandDisplayDevice(d *schema.ResourceData) *computeBeta.DisplayDevice {
+	if _, ok := d.GetOk("enable_display"); !ok {
+		return nil
+	}
+	return &computeBeta.DisplayDevice{
+		EnableDisplay:   d.Get("enable_display").(bool),
+		ForceSendFields: []string{"EnableDisplay"},
+	}
+}
+
+func extractEnableDisplay(displayDevice *computeBeta.DisplayDevice) bool {
```

Comment:
nit: We usually name the complimentary methods `expand` and `flatten`. It's a Terraform convention.